src/styles/components/widget.scss

```diff
@@ -155,12 +167,46 @@ $trix-button--icon-size: 1.6em;
+                      background-image: url('data:image/svg+xml,%3Csvg%20width%3D%2224%22%20height%3D%2224%22%20viewBox%3D%220%200%2024%2024%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Cg%20id%3D%22link-icon%22%3E%0A%3Crect%20id%3D%22Rectangle%2097%22%20width%3D%2224%22%20height%3D%2224%22%20fill%3D%22white%22%2F%3E%0A%3Cpath%20id%3D%22link%22%20d%3D%22M11.154%2015.8529C11.36%2015.6544%2011.4916%2015.4173%2011.5488%2015.1415C11.6117%2014.8658%2011.5946%2014.6011%2011.4973%2014.3474L12.5187%2013.3713C12.8105%2013.8125%2012.9564%2014.3006%2012.9564%2014.8355C12.9564%2015.591%2012.6789%2016.2417%2012.1239%2016.7877L11.1712%2017.6976C10.6047%2018.2325%209.92949%2018.5%209.14557%2018.5C8.36166%2018.5%207.68361%2018.2298%207.11141%2017.6893L5.84113%2016.4651C5.28038%2015.9301%205%2015.2822%205%2014.5211C5%2013.7546%205.28038%2013.1011%205.84113%2012.5607L6.78526%2011.6507C7.35173%2011.1158%208.02693%2010.8483%208.81084%2010.8483C9.37159%2010.8483%209.88371%2010.9917%2010.3472%2011.2785L9.3344%2012.2546C9.15702%2012.1939%208.9825%2012.1636%208.81084%2012.1636C8.39313%2012.1636%208.03837%2012.3042%207.74655%2012.5855L6.80242%2013.4954C6.5106%2013.7767%206.36469%2014.1186%206.36469%2014.5211C6.36469%2014.9182%206.5106%2015.2574%206.80242%2015.5386L8.08129%2016.7629C8.37311%2017.0441%208.72787%2017.1847%209.14557%2017.1847C9.56328%2017.1847%209.91804%2017.0441%2010.2099%2016.7629L11.154%2015.8529ZM18.1663%207.03493C18.727%207.56985%2019.0074%208.22059%2019.0074%208.98713C19.0074%209.74816%2018.727%2010.3989%2018.1663%2010.9393L17.2307%2011.841C16.67%2012.3814%2015.9948%2012.6517%2015.2052%2012.6517C14.6501%2012.6517%2014.1437%2012.511%2013.686%2012.2298L14.6988%2011.2537C14.8819%2011.3088%2015.0507%2011.3364%2015.2052%2011.3364C15.6171%2011.3364%2015.969%2011.1958%2016.2609%2010.9145L17.205%2010.0046C17.4968%209.72335%2017.6427%209.38419%2017.6427%208.98713C17.6427%208.58456%2017.4968%208.24265%2017.205%207.9614L15.9347%206.73713C15.6429%206.45588%2015.2881%206.31526%2014.8704%206.31526C14.447%206.31526%2014.0951%206.45588%2013.8147%206.73713L12.8706%207.64706C12.5788%207.91728%2012.4329%208.25643%2012.4329%208.66452C12.4329%208.82996%2012.4643%208.9954%2012.5273%209.16085L11.5145%2010.1369C11.2169%209.69577%2011.0682%209.20496%2011.0682%208.66452C11.0682%207.90901%2011.3457%207.25827%2011.9007%206.71232L12.8448%205.80239C13.4113%205.26746%2014.0865%205%2014.8704%205C15.6543%205%2016.3295%205.26746%2016.896%205.80239L18.1663%207.03493ZM10.0725%2014.546L14.8275%209.96324L13.8576%209.02849L9.10266%2013.6112L10.0725%2014.546Z%22%20fill%3D%22%23164176%22%2F%3E%0A%3C%2Fg%3E%0A%3C%2Fsvg%3E%0A');
+                    }
                   }
                 }
               }
             }
           }
 
+          /* Tooltip caret */
+          > trix-toolbar:after {
+            content: ' ';
+            display: block;
+            position: absolute;
+            top: 80%;
```

Comment:
Let's use a variable to define the caret radius so that the caret is horizontally centered, fixed to the bottom, and can be made smaller/larger in one place

```sass
          /* Tooltip caret */
          $caret-radius: 5px;
          > trix-toolbar:after {
            content: '';
            display: block;
            position: absolute;
            bottom: -$caret-radius;
            left: calc(50% - #{$caret-radius});
            width: 2 * $caret-radius;
            height: 2 * $caret-radius;
            background: #ffffff;
            border-left: 1px solid $human-grey;
            border-bottom: 1px solid $human-grey;
            -moz-transform: rotate(-45deg);
            -webkit-transform: rotate(-45deg);
          }
```